src/asic/cli.py

```diff
@@ -347,6 +347,7 @@ def download(
 
             preprocessed = f.preprocess(local)
             write_to = preprocessed_path.with_suffix(".csv")
+            os.makedirs(preprocessed_path.parent, exist_ok=True)
```

Comment:
Lets put this behaviour behind a option flag

Reply:
Done